### ISSUE TITLE
this[property] gets recognized as class member in ESDoc.

### ISSUE THREAD

**Issue by Rukenshia** [esdoc/esdoc#59]
I sometimes manipulate properties of my class via this[property] = value. This however generates a public member 'property' in my ESDoc generated document:

```javascript
class Foo {
  /**
   * Creates a new instance of Foo
   *
   * @constructor
   */
  constructor() {
    /** @type {string} */
    this.foo = 'bar';
  }

  /**
   * Set the given property in the class.
   *
   * @param {string} property property to set
   * @param {*} value new value
   */
  manipulate(property, value) {
    this[property] = value;
  }
}
```

![Result](http://puu.sh/jSysV/0cdf5f33cf.png)

**Comment by h13i32maru** [comment]
@Rukenshia Thanks for this issue!
This is ESDoc bug. Please wait a little for fixing it.

**Comment by h13i32maru** [comment]
I published ESDoc v0.2.4. Please try to use it :smile:

**Comment by Rukenshia** [comment]
Works fine now.
